include/tvm/relay/pass.h

```diff
@@ -232,14 +232,8 @@ struct StructuralHash {
  */
 Expr ToANF(const Expr& e, const Module& mod);
 
-inline bool IsPrimitiveFunction(const Function& fn) {
-  NodeRef res = FunctionGetAttr(fn, "Primitive");
-  const ir::IntImm* pval = res.as<ir::IntImm>();
-  return pval && (pval->value != 0);
-}
-
 inline bool IsPrimitiveFunction(const Expr& e) {
-  return e.as<FunctionNode>() && IsPrimitiveFunction(Downcast<Function>(e));
+  return e.as<FunctionNode>() && Downcast<Function>(e)->IsPrimitive();
```

Comment:
move this function to your using side for now to minimize the amount of interface being exposed in include